source/PreferencesPanel.cpp

```diff
@@ -140,6 +140,8 @@ void PreferencesPanel::Draw()
 
 	Information info;
 	info.SetBar("volume", Audio::Volume());
+	info.SetBar("music volume", Audio::MusicVolume());
+	info.SetCondition("show music slider");
```

Comment:
What's the point of this condition if it's always set?

Reply:
removed

Reply:
The point of the condition is to hide the slider if there is no music.  The original thought is why show a slider that does nothing from a gamer perspective.

Does the music slider remain hidden if there is no music?

Reply:
-> no as said by @Amazinite here: https://github.com/endless-sky/endless-sky/pull/8491#discussion_r1438920488

Reply:
I replied to them as well.  I think it will be odd to have a slider that does nothing in the UI but if you both are adamant about it I don't plan to revert it.

Currently, (and I don't think there will be any time soon) content creators have no plans to integrate music. I shopped the idea often when I had the plugin working and there wasn't any interest from the vanilla content creators.

Meaning vanilla is going to have a volume slider that doesn't do anything.

Reply:
Doesn't the new slider also control ambient system music and the background noises that play when you land on a station?

Reply:
It's not the same.  It controls the playlist tracks.  As far as I know the sounds slider still controls all of the old definitions of sound.